modules/account-lib/src/coin/dot/transactionBuilder.ts

```diff
@@ -194,9 +194,10 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
     if (utils.isSigningPayload(decodedTxn)) {
       this.referenceBlock(decodedTxn.blockHash);
     } else {
-      this.sender({ address: utils.decodeDotAddress(decodedTxn.address) });
+      const keypair = utils.decodeDotAddressToKeyPair(decodedTxn.address);
```

Comment:
ditto: reasons for changes

Reply:
BuildImplementation sign a transaction when signature is added in transactionBuilder. 
Two line below, it changes from directly push the signature in the transaction to addSignature in the builder.
AddSignature requieres the public key. This changes is to get address and public key in one step.

https://github.com/BitGo/BitGoJS/blob/bc58eda7da65815cd79521b103e5584e82d84298/modules/account-lib/src/coin/dot/transactionBuilder.ts#L224-L227